test/elixir_tic_tac_toe_basic_test.exs

```diff
@@ -18,12 +22,22 @@ defmodule ElixirTicTacToeBasicTest do
     end
   end
 
+  defmodule TestOutcomeChecker do
+    def check_full(_config) do
+      Helpers.Stack.pop()
```

Comment:
**issue (blocking)**: Let's create a test double that does not use `Helpers.Stack`

I'd like to see you explore creating a double that doesn't rely on statefulness `Helpers.Stack`. What would that look like? While we did use `Helpers.Stack` elsewhere, it still violates the spirit of one of our constraints.